include/rdma/fabric.h

```diff
@@ -315,6 +315,21 @@ enum {
 	FI_PROTO_EFA
 };
 
+#define FI_TC_PROV_SPECIFIC (1UL << 31)
+
+enum {
+	FI_TC_UNSPEC = 0,
+	FI_TC_DSCP = FI_TC_UNSPEC,
+	FI_TC_LABEL = 0x100,
+	FI_TC_BEST_EFFORT = FI_TC_LABEL,
+	FI_TC_LOW_LATENCY,
+	FI_TC_DEDICATED_ACCESS,
+	FI_TC_BULK_DATA,
+	FI_TC_SCAVENGER,
+	FI_TC_NETWORK_CTRL,
+	FI_TC_REALTIME_STREAM, /* TODO: pending description @shefty */
```

Comment:
Realtime stream maps to video streams, where the data needs dedicated bandwidth and little jitter.